pkg/chartutil/create.go

```diff
@@ -210,8 +231,17 @@ spec:
       labels:
         {{- include "<CHARTNAME>.selectorLabels" . | nindent 8 }}
     spec:
+    {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      serviceAccountName: {{ template "<CHARTNAME>.serviceAccountName" . }}
```

Comment:
`include` is used in all other places. Let's be consistent.

```suggestion
      serviceAccountName: {{ include "<CHARTNAME>.serviceAccountName" . }}
```